src/main.ts

```diff
@@ -58,7 +126,7 @@ async function guardedRun(): Promise<void> {
     return;
```

Comment:
I'm unable to leave a comment on line 122 (`const sccacheDirectory = find...`), but I'm not sure how to proceed if the `VERSION` constant. If we call `getLatestRelease` to get the version, and the release is published but prebuilt binaries are not available, an error will be thrown. Should we do [`findAllVersions`](https://github.com/actions/toolkit/blob/819157bf872a49cfcc085190da73894e7091c83c/packages/tool-cache/src/tool-cache.ts#L512) instead and choose the latest available version?

Reply:
They released 0.3.3, so we can put it now.